src/constraints/NumberConstraints.ts

```diff
@@ -98,3 +98,21 @@ export const numberNeNaN: IConstraint<number> = {
 			: Result.ok(input);
 	}
 };
+
+export function numberDivisibleBy(divider: number): IConstraint<number> {
+	const expected = `% ${divider}`;
+	return {
+		run(input: number) {
+			return input % divider === 0 //
+				? Result.ok(input)
+				: Result.err(
+						new ConstraintError(
+							'numberDivisibleBy',
+							`Expected number to be divisible by ${divider}, but received ${input}`,
+							input,
+							expected
+						)
+				  );
```

Comment:
I spy with my little eye and I see mixed tabs and spaces

![image](https://user-images.githubusercontent.com/4019718/151200591-1026355b-9881-4682-bbf9-d1432af2d655.png)

Reply:
Do you seriously think I have the time to set 2 spaces after a bunch of tabs?

That is obviously prettier's doing